viper/parser.py

```diff
@@ -244,12 +288,12 @@ def parse_tree_to_lll(code, origcode):
 
 # Checks that an input matches its type
 def make_clamper(datapos, mempos, typ, is_init=False):
-    if not is_init:
```

Comment:
What is the reason for removing the `codeload`? This appears to break functionality. If I add the two if-branches back in again, the tests pass.